src/libpriv/rpmostree-scripts.c

```diff
@@ -227,60 +292,18 @@ run_known_rpm_script (const KnownRpmScriptKind *rpmscript,
   if (!script)
```

Comment:
Hmm...it is a bit of duplication, but on the other hand as written `run_script()` is detecting whether or not we need to do work, then when we go to run we know there's a script.  It's a free constant access anyways.